system/ssl/trusted_windows.go

```diff
@@ -18,7 +18,7 @@ func AddTrustedCertToKeychain(certificate string) {
 
 // RemoveTrustedCertFromKeychain ...
 func RemoveTrustedCertFromKeychain(certificate string) {
-	fmt.Println("The generated SSL certificates need to be manually removed from your keychain if they were added during `tok up`. See https://tokaido.io/docs/config/#adding-a-trusted-certificate for more information.")
+	fmt.Println("    The generated SSL certificates need to be manually removed from your keychain if they were added during `tok up`. \n    See https://tokaido.io/docs/config/#adding-a-trusted-certificate for more information.")
```

Comment:
If these certs messages are the same for each platform, could probably be a const in the `constants` package?

Reply:
I thought about that, but they won't be long term. Once we have the new site up and running, I plan to add doc sections for updating SSL certs on each platform, so these will actually link to different places.